main.py

```diff
@@ -210,6 +210,10 @@ async def ws_type_command(db, websocket, data):
             sw.on()
         else:
             sw.off()
+    if thing.type in ["shellytrv"]:
```

Comment:
This should be an `elif` as otherwise `thing.type` being one of `'switch', 'shelly'` would go through the `if` above and trigger the `else` now attached to this `if`.